be/src/common/logconfig.cpp

```diff
@@ -249,7 +249,7 @@ bool init_glog(const char* basename, bool install_signal_handler) {
 
     if (config::dump_trace_info) {
         google::InstallFailureWriter(failure_writer);
-        google::InstallFailureFunction(failure_function);
+        google::InstallFailureFunction((google::logging_fail_func_t)failure_function);
```

Comment:
why add this cast?

Reply:
make clang-tidy happy.

Reply:
![Uploading img_v3_02d5_1ebd0b74-c54c-4b00-a83c-deb84fe3374g.jpg…]()

Reply:
![img_v3_02d5_1ebd0b74-c54c-4b00-a83c-deb84fe3374g](https://github.com/user-attachments/assets/697fdfd2-0047-44bd-9353-2da24c4754d3)